crates/sp-executor/src/lib.rs

```diff
@@ -128,23 +142,32 @@ pub struct SignedBundle<Extrinsic> {
 
 /// Bundle with opaque extrinsics.
 #[derive(Debug, Decode, Encode, TypeInfo, PartialEq, Eq, Clone)]
-pub struct OpaqueBundle {
+pub struct OpaqueBundle<Number, Hash, SecondaryHash> {
```

Comment:
Hm... I thought the idea of `OpaqueBundle` was that is it opaque, meaning doesn't need to be generic and can just be sent throgh as is. Now it is basically the same as `Bundle<Extrinsic = OpaqueExtrinsic, Number, Hash, SecondaryHash>`. So maybe we don't need a separate type at all and just reuse the same type with a different parameter (or make it a partially specialized type alias)?

Reply:
>I thought the idea of OpaqueBundle was that is it opaque, meaning doesn't need to be generic and can just be sent throgh as is.

Yeah, it was true in the ancient time, now a lot of things have been changed, we can consider removing the OpaqueBundle, will have a look in the next PR. The executor tests seem to fail more frequently recently :disappointed: .